### PR TITLE
Using official Oracle JDBC driver from maven central

### PR DESCRIPTION
**Pull request by usmansaleem**
Signed-off-by: Usman Saleem <usman@usmans.info>

### CODE REVIEW

File: gradle/check-licenses.gradle [hunk 1]
```diff
@@ -79,8 +79,8 @@ downloadLicenses {
     'The MIT License',
     'https://opensource.org/licenses/MIT')
   ext.otn = license(
-    'Oracle Technology Network License Agreement - Oracle Maven Repository',
-    'https://www.oracle.com/webapps/maven/register/license.html'
+    'Oracle Free Use Terms and Conditions (FUTC)',
+    'https://repo1.maven.org/maven2/com/oracle/ojdbc/ojdbc10/19.3.0.0/ojdbc10-19.3.0.0.pom'
```
Comment (shemnon) [inline gradle/check-licenses.gradle]: Is there a better location than the POM of a specific version of OJDBC for this text?

Comment (usmansaleem) [reply gradle/check-licenses.gradle]: Unfortunately not, I tried google search for it and it is not available anywhere at the moment (except inside this pom file).

Comment (usmansaleem) [reply gradle/check-licenses.gradle]: @shemnon should we host it somewhere as a gist or in our repo as txt/html file??

Comment (shemnon) [reply gradle/check-licenses.gradle]: Hosting it in a gist is an awesomely hacky solution.  Lets to that.  Or some other URL that will give the raw text.  The reason we need the raw text is for automatic license notificaitons.

Comment (usmansaleem) [reply gradle/check-licenses.gradle]: @shemnon The only issue with gist is that it contains git author user id in the url.

Comment (usmansaleem) [reply gradle/check-licenses.gradle]: @shemnon  Are you ok with using this link? [](https://github.com/PegaSysEng/orion/wiki/Oracle-JDBC-Driver-License---Oracle-Free-Use-Terms-and-Conditions-(FUTC).md)

Comment (shemnon) [reply gradle/check-licenses.gradle]: It chops off newlines.  And it's mutable.  a gist to a specific version would be better.

Comment (ajsutton) [reply gradle/check-licenses.gradle]: The pegasus-admin GitHub user could create a gist if that helps.

Comment (usmansaleem) [reply gradle/check-licenses.gradle]: @ajsutton yes, that would work. I'll coordinate with @joshuafernandes to create the gist as a .txt file.

Comment (usmansaleem) [reply gradle/check-licenses.gradle]: @shemnon @ajsutton https://gist.githubusercontent.com/pegasys-admin/54b5ab9e846d0f90429ddf4881214d5e/raw/4e038c4302679956f8dfe6d5bf5de183c9af7054/oracle_futc.txt

Comment (shemnon) [reply gradle/check-licenses.gradle]: Looks good, it won't accidentally change.

Comment (shemnon) [reply gradle/check-licenses.gradle]:
```suggestion
    'https://gist.githubusercontent.com/pegasys-admin/54b5ab9e846d0f90429ddf4881214d5e/raw/4e038c4302679956f8dfe6d5bf5de183c9af7054/oracle_futc.txt'
```

**Pull request merged**